### PR TITLE
Ys/changes

### PR DESCRIPTION
**Pull request by ysadka**
styling, analytics add, residential login link

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -27,5 +27,14 @@ module.exports = {
       },
     },
     'gatsby-plugin-offline',
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: 'UA-139694645-1',
+        head: true,
+        anonymize: true,
+        respectDNT: true,
```
Comment (jonsadka) [inline gatsby-config.js]: Probably want to set these to false

File: package.json [hunk 2]
```diff
@@ -4,22 +4,24 @@
   "version": "1.0.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
-    "babel-plugin-styled-components": "^1.8.0",
+    "babel-plugin-styled-components": "^1.10.0",
     "eslint": "^5.9.0",
     "gatsby": "^2.0.45",
+    "gatsby-cli": "^2.5.12",
```
Comment (jonsadka) [inline package.json]: What is this used for?

File: src/components/footer.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const FooterContent = styled.div`
 
 const Nav = () => (
   <FooterWrapper>
-    <FooterContent>© 2018</FooterContent>
+    <FooterContent>© 2019</FooterContent>
```
Comment (jonsadka) [inline src/components/footer.js]: FYI we can just change this to be `{new Date().getFullYear()}`

File: package.json (hunk 2 above)
Comment (ysadka) [reply package.json]: wanted me to install it to run gatsby commands

File: gatsby-config.js (hunk 1 above)
Comment (ysadka) [reply gatsby-config.js]: changed head but respect privacy for other stuff

**Pull request merged**